titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/admission/JobImageSanitizer.java

```diff
@@ -95,7 +95,8 @@ private boolean isDisabled() {
 
     // Determines if this Exception should produce a sanitization failure
     private boolean isValidationOK(Throwable throwable, Image image) {
```

Comment:
Not part of your changes, but should probably be renamed to `isValidationExceptionOK()`. The method name is somewhat confusing.

Reply:
+1

Reply:
Fixed.